suricata/update/main.py

```diff
@@ -193,14 +198,14 @@ def fetch(self, url):
             tmp_fileobj.close()
         except URLError as err:
             if os.path.exists(tmp_filename):
-                logger.warning(
+                strict_error(
                     "Failed to fetch %s, "
                     "will use latest cached version: %s", url, err)
```

Comment:
I think we need a conditional here and log differently.  If not strict, the message is valid as we'll use the latest cached version.  If strict we're not going to use the cached version as its a fatal error.